lucetc/lucetc/options.rs

```diff
@@ -188,6 +238,69 @@ impl Options {
                     .multiple(false)
                     .help("output destination, defaults to a.out if unspecified"),
             )
+            .arg(
+                Arg::with_name("target-cpu")
+                    .long("--target-cpu")
+                    .takes_value(true)
+                    .multiple(false)
+                    .number_of_values(1)
+                    .possible_values(&[
+                        "baseline",
+                        "nehalem",
+                        "haswell",
+                        "broadwell",
+                        "skylake",
+                        "cannonlake",
+                        "icelake",
+                        "znver1",
+                    ])
+                    .help("Generate code for a particular type of CPU.")
+                    .long_help(
+"Generate code for a particular type of CPU.
+
+If neither `--target-cpu` nor `--target-feature` is provided, `lucetc`
+will automatically detect and use the features available on the host CPU.
+
+"
+                    )
+            )
+            .arg(
+                Arg::with_name("target-feature")
+                    .long("--target-feature")
+                    .takes_value(true)
+                    .multiple(true)
+                    .use_delimiter(true)
+                    .possible_values(&[
+                        "+sse3", "-sse3",
+                        "+ssse3", "-ssse3",
+                        "+sse41", "-sse41",
+                        "+sse42", "-sse42",
+                        "+popcnt", "-popcnt",
+                        "+avx", "-avx",
+                        "+bmi1", "-bmi1",
+                        "+bmi2", "-bmi2",
+                        "+lzcnt", "-lzcnt",
```

Comment:
you: "this doesn't look like valid rustfmt"
me: "correct, for some reason rustfmt does nothing to the right-hand side of this `let m = ...` binding. maybe it exceeds some expression size threshold?"